common/src/test/java/tachyon/security/LoginModuleTest.java

```diff
@@ -55,4 +64,47 @@ public void simpleLoginTest() throws Exception {
   }
 
   // TODO: Kerberos login test
+
+  /**
+   * Test whether we can get login user with conf in SIMPLE mode.
+   * @throws Exception
+   */
+  @Test
+  public void getSimpleLoginUserTest() throws Exception {
+    TachyonConf conf = new TachyonConf();
+
+    // set conf to "SIMPLE" and get login user
+    conf.set(Constants.TACHYON_SECURITY_AUTHENTICATION, "SIMPLE");
+    UserInformation.setTachyonConf(conf);
+
+    UserInformation loginUser = UserInformation.getTachyonLoginUser();
+
+    Assert.assertFalse(loginUser.getUserName().isEmpty());
+  }
+
+  /**
+   * Test initializing with conf in NOSASL mode.
+   * @throws Exception
+   */
+  @Test
+  public void initConfTest() throws Exception {
+    // throw exception when initializing conf with "NOSASL"
+    TachyonConf conf = new TachyonConf();
```

Comment:
Done. The class and method names are changed in latest patch. Please refer to `UserInformation.securityEnabledTest()`